Link to DockerHub for upgrade instructions. (#2214)
influxdata/docs-v2: content/influxdb/v2.0/upgrade/v1-to-v2/_index.md

{{% note %}}
#### Docker users

> We have implemented as separate upgrade process to ensure a smooth transition from InfluxDB 1.x to InfluxDB 2.0 on Docker.
> See the "Upgrading from InfluxDB 1.x" section in the [DockerHub docs](https://hub.docker.com/_/influxdb) for instructions and examples.

Reviewer: ```suggestion
To upgrade from InfluxDB 1.x to InfluxDB 2.0 on Docker, see  "Upgrading from InfluxDB 1.x" in the [Docker Hub documentation](https://hub.docker.com/_/influxdb) for instructions and examples.
```
